pkg/lockfile/dpkg-status.go

```diff
@@ -0,0 +1,105 @@
+package lockfile
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+)
+
+const DebianEcosystem Ecosystem = "Debian"
+
+func groupDpkgPackageLines(scanner *bufio.Scanner) [][]string {
+	var groups [][]string
+	var group []string
+
+	for scanner.Scan() {
+		line := scanner.Text()
+
+		if line == "" {
+			if len(group) > 0 {
+				groups = append(groups, group)
+			}
+			group = make([]string, 0)
+
+			continue
+		}
+		group = append(group, line)
+	}
+
+	if len(group) > 0 {
+		groups = append(groups, group)
+	}
+
+	return groups
+}
+
+func parseDpkgPackageGroup(group []string, pathToLockfile string) PackageDetails {
+	var pkg = PackageDetails{
+		Ecosystem: DebianEcosystem,
+		CompareAs: DebianEcosystem,
+	}
+
+	// TODO File SPECS:
```

Comment:
Can you also expand on the implication of not supporting this in this TODO comment? e.g. are there going to be package names we miss from not supporting this?

Reply:
My mistake, the comment is incomplete. I wanted to add link to `status` file format and it was difficult to find. I will update the comment with a useful link